sql/telemetry_derived/ssl_ratios_v1/metadata.yaml

```diff
@@ -6,7 +6,8 @@ owners:
   - chutten@mozilla.com
 labels:
   application: firefox
-  incremental: true
+  incremental: false
   schedule: daily
   public_json: true
   public_bigquery: true
+  review_link: https://bugzilla.mozilla.org/show_bug.cgi?id=1414839
```

Comment:
URLs contain characters that aren't allowed for label keys or values. We could have a simple bug number here instead.

Reply:
Perhaps we call it `review_bug`

Reply:
I guess that is okay for now. Though is it possible in the future that data reviews will be posted on somewhere other than Bugzilla?

Reply:
Indeed that may happen in the future, but it seems like it would be pretty easy to refactor the labels at that point. I'm open to suggestions on a more adaptable format, but don't want to block your work here on it.

Reply:
I think its a good option for now. The code itself should be easy to refactor if/when the use case arises.